File: labs/J10_Observer/python/complete/logger.py

class Logger():
    def warn(self,message):
        print("[log:\033[93m{}\033[00m]".format(message))
    def info(self,message):
        print("[log:\033[92m{}\033[00m]".format(message))
    def error(self,message):
        print("[log:\033[91m{}\033[00m]".format(message))

File: labs/J10_Observer/java/complete/app/src/main/java/ObserverComplete/Logger.java
package ObserverComplete;

public class Logger {

  public Logger() {

  }
  
  public void warn(String message) {
      System.out.println("[Warn] " + message);
  }

  public void info(String message) {
      System.out.println("[Info] " + message);
  }

  public void error(String message) {
      System.out.println("[Error] " + message);
  }

}
